tests/conftest.py

```diff
@@ -1282,3 +1282,9 @@ def duts_minigraph_facts(duthosts, tbinfo):
         }
     """
     return duthosts.get_extended_minigraph_facts(tbinfo)
+
+@pytest.fixture(scope="module", autouse=True)
+def get_reboot_cause(duthost):
+    yield
+    reboot_cause_history_got = duthost.show_and_parse("show reboot-cause history")
+    logger.info("dut {} reboot-cause history {}".format(duthost.hostname, reboot_cause_history_got))
```

Comment:
If purpose of this fixture is just to log the reboot-cause history, then simply run `duthosts.shell('show reboot-cause history')` should be enough. Execution result of this command will be logged at DEBUG level. It's unnecessary to explicitly log it again.
 
And it would be better to run the command on all DUT hosts using the `duthosts` fixture.

Reply:
I will fix it